images/upload-gitlab-failure-logs/upload_gitlab_failure_logs.py

```diff
@@ -196,16 +257,9 @@ def main():
     # Assign any/all relevant errors
     assign_error_taxonomy(job_input_data, job_trace)
 
-    # Upload to OpenSearch
-    connections.create_connection(
-        hosts=[OPENSEARCH_ENDPOINT],
-        http_auth=(
-            OPENSEARCH_USERNAME,
-            OPENSEARCH_PASSWORD,
-        ),
-    )
-    doc = JobPayload(**job_input_data)
-    doc.save()
+    # Push data to outputs
+    push_record_to_opensearch(job_input_data)
```

Comment:
Just to clarify, the plan is to leave opensearch uploads in place for now until we have full confidence in the postgres replication, and then drop it, right?

Reply:
Correct, just playing it safe